### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by monicah79**
- when i run the rails routes command in the console, you can see the following output:
4d190ddc-e5d6-40f7-b550-cab2c2606da4
- For each URL  created:
   * A route.
   * An action in the correct controller.
   * A view file.
- View files include only placeholders written in plain HTML,
      *Here is a list of posts for a given user
below are the URLs
http://localhost:3000/post/show
http://localhost:3000/post/index
http://localhost:3000/user/show
http://localhost:3000/user/index

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'post/index'
```
Comment (Davidosky007) [inline config/routes.rb]: * When i ran the `rails routes` command i didn't get the expected results kindly look into it 
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @monicah79 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Controllers used to handle requests and render empty views 👍 
* Well written PR 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (monicah79) [reply config/routes.rb:2]: Hey here is my screenshoot
image

Comment (monicah79) [reply config/routes.rb]: image

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: :raising_hand: @monicah79, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

:tada: :champagne:  :tada: :champagne: 

## Highlights :green_circle: 

- Unnecessary endpoints are not exposed :+1: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**